how-to/rpi-camera.rst

```diff
@@ -144,10 +145,37 @@ You can follow it to learn how to use ``picamera2`` on Ubuntu.
 
 .. important::
     Chapter 9.8 and 9.9 of the ``picamera2`` manual linked above make use of the Hailo and IMX500 AI accelerator. The Hailo
-    AI accelerator is not supported on Ubuntu, and the IMX500 AI accelerator requires the imx500-firmware package, which
+    AI accelerator is not supported on Ubuntu, and the IMX500 AI accelerator requires the ``imx500-firmware`` package, which
     can't be included in Ubuntu due to licensing issues. So anything in those chapters might not work out of the box.
     We are working on fixing the licensing issues with the involved parties.
 
+Workaround for the IMX500 (AI Camera)
+=====================================
+
+.. important::
+    While the steps given below will allow you to use the IMX500 camera module on Ubuntu,  the ``imx500-firmware``
```

Comment:
Minor change here: the style-guide requests we only use the ["warning" and "information" admonitions](https://docs.ubuntu.com/styleguide/en#admonitions); this should probably be "warning" (information is generally "note" in reST as this is in the base specification and not a Sphinx add-on)